SWLOR.Game.Server/Feature/LootTableDefinition/HutlarLootTableDefinition.cs

```diff
@@ -89,5 +91,25 @@ private void QionTigers()
                 .AddItem("tiger_meat", 5)
                 .AddItem("q_tiger_paw", 2, 1, true);
         }
+
+        private void FrostSaber()
+        {
+            _builder.Create("HUTLAR_FROST_SABER")
+                .AddItem("lth_good", 20)
+                .AddItem("f_saber_fur", 10)
```

Comment:
I'll need to review the items you've added here. We'll have to get them integrated into the crafting system too.

Reply:
All of them have no stats at the moment but wasn't sure what item size and drop rates should be or if that's too much loot for a single critter.